test/bdd/features/search.feature

```diff
@@ -0,0 +1,59 @@
+# Proposed by Bello
+Feature: Searching for assets on the DKG
+
+    Scenario: Search for an entity with an existing keyword
+        Given I search for an entity with one of its existing keywords:
+            | keyword 1 |
+        When I call entities search result with the given handler id
+        Then I will get the assertion id for the matching dataset
+        And If I call resolve route with the assertion id
+        Then I will get the handler id for the dataset
+        And If I call resolve result with the handler id
+        Then I will get the dataset containing the intial keyword:
```

Comment:
Check the following:

Number of nodes that responded
Number of assets retrieved